tests/test_integration/test_scheduler/test_scheduler.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+
+import pytest
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from syncmaster.db.models import Run, Status
+from syncmaster.scheduler import TransferFetcher, TransferJobManager
+from syncmaster.settings import Settings
+from tests.mocks import MockTransfer
+
+pytestmark = [pytest.mark.asyncio, pytest.mark.worker, pytest.mark.scheduler_integration]
+
+
+async def test_scheduler(
+    session: AsyncSession,
+    settings: Settings,
+    group_transfer: MockTransfer,
+    transfer_job_manager: TransferJobManager,
+    mock_send_task_to_tick,
+    mock_add_job,
+):
+    transfer_fetcher = TransferFetcher(settings)
+    transfers = await transfer_fetcher.fetch_updated_jobs()
+    assert transfers
+    assert group_transfer.transfer.id in [t.id for t in transfers]
```

Comment:
nit: use set instead of list if you need `item in container` check

Reply:
there are only 6 entries, so I thought it will not give an increase, but we still can use set